src/notation/qml/MuseScore/NotationScene/internal/EditStyle/TiePlacementSelector.qml

```diff
@@ -70,12 +70,13 @@ Rectangle {
 
         RadioButtonGroup {
             model: [
-                { iconCode: IconCode.TIE_CHORD_INSIDE, value: 1, title: qsTrc("inspector", "Inside") },
+				{ iconCode: IconCode.AUTO_TEXT,         value: 0, title: qsTrc("inspector", "Auto") },
```

Comment:
We prefer using spaces instead of tabs for indentation (that would also fix the formatting on GitHub)